include/flashinfer/attention/decode.cuh

```diff
@@ -439,6 +439,9 @@ __global__ void BatchDecodeWithPagedKVCacheKernel(const __grid_constant__
   vec_t<float, vec_size> q_vec;
   vec_t<float, vec_size> freq;
   int32_t q_offset_val = q_offset == nullptr ? (kv_len - 1) : q_offset[batch_idx];
+  // TODO(cilei): set q_vec according to stride values from params
+  const uint32_t q_stride_n = params.q_stride_n;
```

Comment:
remove const, since the Line 441 does not have 'const'

Reply:
But L412-L415 all have `const`

Reply:
My bad, we should refactor to follow some unified convention some time :)